cmd/init.go

```diff
@@ -20,6 +21,59 @@ const (
 	DefaultFolderName = "bruin-pipeline"
 )
 
+var choices = templates.TemplatesName()
+
+type model struct {
+	cursor int
+	choice string
+}
+
+func (m model) Init() tea.Cmd {
+	return nil
+}
+
+func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
+	if msg, ok := msg.(tea.KeyMsg); ok {
+		switch msg.String() {
+		case "ctrl+c", "q", "esc":
+			return m, tea.Quit
+		case "enter":
+			// Send the choice on the channel and exit.
+			m.choice = choices[m.cursor]
+			return m, tea.Quit
+		case "down", "j":
+			m.cursor++
+			if m.cursor >= len(choices) {
+				m.cursor = 0
+			}
+		case "up", "k":
+			m.cursor--
+			if m.cursor < 0 {
+				m.cursor = len(choices) - 1
+			}
+		}
+	}
+	// Return the concrete type instead of the interface
+	return m, nil // This line is fine as it is
+}
+func (m model) View() string {
+	s := strings.Builder{}
+	s.WriteString("Please select the template?\n\n")
+
+	for i, choice := range choices {
+		if m.cursor == i {
+			s.WriteString("(•) ")
+		} else {
+			s.WriteString("( ) ")
```

Comment:
I would indent these a bit:
```suggestion
			s.WriteString(" [x] ")
		} else {
			s.WriteString(" [ ] ")
```